lib/matplotlib/tests/test_axes.py

```diff
@@ -75,7 +75,7 @@ def test_repr():
         "label='label', title={'center': 'title'}, xlabel='x', ylabel='y'>")
 
 
-@check_figures_equal()
+@check_figures_equal(extensions=['png'])
```

Comment:
Why does this change test decorators?

Reply:
The backport of the #29827 did not include the change of defaults to `check_figures_equal` to only build `png` by default for API stability reasons. Thus the extensions remained explicit on the `v3.10.x`.

So any `extensions=['png']` being added here that correspond with https://github.com/matplotlib/matplotlib/pull/29827/commits/4b51837f3161d65e9cf7ccf5f7d95562d71163e8 can be reverted.